blogposts/2021/dev-tool-time-paul-jolly.md

```diff
@@ -0,0 +1,86 @@
+---
+title: '“It’s small things that, when combined, make your development experience that much more pleasant”: Dev Tool Time with Paul Jolly'
+description: 'Paul Jolly, CTO and founder at myitcv.io, shares how a minimalist desk setup, combined with the sophisticated programming language CUE, creates a streamlined development environment.'
+author: Nick Moore
+authorUrl: https://twitter.com/nickwritesit
+publishDate: 2021-09-22T10:00-07:00
+tags: [blog]
+slug: dev-tool-time-paul-jolly
+heroImage:
+socialImage:
+published: true
+---
+
+When Paul Jolly, CTO and founder at [myitcv.io](https://myitcv.io/), joined us for Dev Tool Time on September 15, 2021, he toured us through a workflow that included an eight-year-old laptop, excluded a chair, and involved extensive use of CUE, an open source data validation language.
+
+You can watch the recording of this episode at the bottom of this blog post, but first let’s check out Paul’s desk:
+
+![Paul’s work desk](https://storage.googleapis.com/sourcegraph-assets/blog/dev-tool-time/dev-tool-time-paul-jolly-desk-big.jpeg)
+
+## Hardware setup
+
+Paul called his setup “very uninteresting.” His monitor, perched atop a thick dictionary, oversaw what he called a “standard home office setup.” Not pictured is his chair–because he doesn’t have one. Thanks to a standing desk, Paul said, “I haven’t had a seat in years.”
+
+Paul’s main computer is a late 2013 Macbook Pro that comes with–he’s careful to note–real-life function keys: “No touchbar for me.” He also uses a thunderbolt display, an external Apple keyboard, a touchpad, and a basic Microsoft mouse.
+
+In terms of hardware, Paul likes stability. The real magic, as we discovered later in the call, happens in CUE.
+
+## Linux and VMware Fusion
+
+Like Mitchell Hashimoto, [who we hosted in August](https://about.sourcegraph.com/blog/dev-tool-time-mitchell-hashimoto/), Paul does most of his development within Linux, which he runs within VMware Fusion. In terms of disk i/o, development within a VMware Fusion virtual machine, running on top of MacOS, is actually [faster than native Mac](https://www.google.com/url?q=https://github.com/golang/go/issues/28739&sa=D&source=editors&ust=1631903935728000&usg=AOvVaw3lB31d39lsuwMyfXtjHtph).
+
+Paul runs Ubuntu 21.04, though notes he is “very taken” by Mitchell’s [NixOS setup](https://about.sourcegraph.com/blog/dev-tool-time-mitchell-hashimoto/#NixOS). Instead of Nix snapshots, he uses VMware Fusion snapshots, which he acknowledges are coarser. Paul is also a heavy user of [AwesomeWM](https://awesomewm.org/), though notes he might switch to [i3](https://i3wm.org/) one day.
```

Comment:
Consider changing the wording of one of your "though notes" occurrences, to reduce repetition.